src/test/java/by/itechart/api/service/impl/UserServiceImplTest.java

```diff
@@ -85,8 +89,9 @@ void testDeleteExistingUser() {
     @DisplayName("Find all users")
     void testFindAllUsers() {
         List<User> suggestedAllUsers = List.of(new User(), new User());
-        when(userRepository.findAll()).thenReturn(suggestedAllUsers);
-        List<UserDTO> resultAllUsers = userService.findAll();
+        Page<User> pages = new PageImpl<>(suggestedAllUsers);
+        when(userRepository.findAll(PageRequest.of(1, 10, Sort.by("firstName")))).thenReturn(pages);
+        List<UserDTO> resultAllUsers = userService.findAll(PageRequest.of(1, 10, Sort.by("firstName")));
         assertThat(resultAllUsers).isNotEmpty().size().isEqualTo(2);
```

Comment:
I think `isNotEmpty()` check is redundant since you are validating size. You can also use just `hasSize()`